lib/tmuxinator/config.rb

```diff
@@ -14,12 +14,11 @@ def sample
         asset_path "sample.yml"
       end
 
-      def default
-        "#{ENV['HOME']}/.tmuxinator/default.yml"
-      end
-
-      def default?
-        exists?("default")
+      def default(filename)
+        if exists? (filename)
```

Comment:
For consistency with the rest of the project, there should be no space after the `?`.